openssl-sys/src/ossl110.rs

```diff
@@ -0,0 +1,142 @@
+use libc::{c_int, c_void, c_char, c_uchar, c_ulong, c_long};
+
+pub enum stack_st_X509 {}
+pub enum stack_st_X509_ATTRIBUTE {}
+pub enum stack_st_X509_EXTENSION {}
+pub enum stack_st_GENERAL_NAME {}
+pub enum stack_st_void {}
+pub enum _STACK {}
+pub enum BIO_METHOD {}
+pub enum RSA {}
+pub enum DSA {}
+pub enum EVP_PKEY {}
+pub enum BIO {}
+pub enum CRYPTO_EX_DATA {}
+pub enum EVP_MD_CTX {}
+pub enum EVP_CIPHER {}
+pub enum HMAC_CTX {}
+pub enum BIGNUM {}
+pub enum OPENSSL_STACK {}
+pub enum DH {}
+pub enum X509 {}
+pub enum SSL_CTX {}
+
+pub const SSL_OP_MICROSOFT_SESS_ID_BUG: c_ulong =                   0x00000000;
+pub const SSL_OP_NETSCAPE_CHALLENGE_BUG: c_ulong =                  0x00000000;
+pub const SSL_OP_NETSCAPE_REUSE_CIPHER_CHANGE_BUG: c_ulong =        0x00000000;
+pub const SSL_OP_MICROSOFT_BIG_SSLV3_BUFFER: c_ulong =              0x00000000;
+pub const SSL_OP_SSLEAY_080_CLIENT_DH_BUG: c_ulong =                0x00000000;
+pub const SSL_OP_TLS_D5_BUG: c_ulong =                              0x00000000;
+pub const SSL_OP_TLS_BLOCK_PADDING_BUG: c_ulong =                   0x00000000;
+pub const SSL_OP_SINGLE_ECDH_USE: c_ulong =                         0x00000000;
+pub const SSL_OP_SINGLE_DH_USE: c_ulong =                           0x00000000;
+pub const SSL_OP_NO_SSLv2: c_ulong =                                0x00000000;
+
+pub const OPENSSL_VERSION: c_int = 0;
+pub const OPENSSL_CFLAGS: c_int = 1;
+pub const OPENSSL_BUILT_ON: c_int = 2;
+pub const OPENSSL_PLATFORM: c_int = 3;
+pub const OPENSSL_DIR: c_int = 4;
+
+pub const CRYPTO_EX_INDEX_SSL: c_int = 0;
+pub const CRYPTO_EX_INDEX_SSL_CTX: c_int = 1;
+
+pub fn init() {
+    // TODO: fill this in, probably need to at least do something?
```

Comment:
Oh yay! Sounds like we don't actually have to do anything in that case, thanks @jwilm!